packages/telemetry/src/ingest-slog-entrypoint.js

```diff
@@ -74,6 +74,7 @@ async function run() {
     if (!flush) {
       return;
     }
+    assert(slogSender.forceFlush, 'forceFlush missing from slogSender');
     await slogSender.forceFlush();
```

Comment:
I think it'd be more correct to conditionally call.

```suggestion
    await slogSender.forceFlush?.();
```

Reply:
thanks. at the point I wrote this it seemed like `forceFlush` wasn't truly optional but I see now it is.